en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md

```diff
@@ -0,0 +1,286 @@
+                   <property name="Delete selected container" value="Delete selected container"/>
+               </log>
+               <respond/>
+           </inSequence>
+           <outSequence/>
+           <faultSequence/>
+       </resource>
+   </api>
+   ```
+**Note**
+
+* As `accountKey` use the access key obtained from setting up the Microsoft Azure Storage account.
+* As `accountName` get the name of created **Storage Account** inside the Microsoft Azure Storage account.
+    
+Now we can export the imported connector and the API into a single CAR application. CAR application is the one we are going to deploy to server runtime. 
+   
+{!/references/connectors/exporting-artifacts.md!}
+
+## Get the project
+
+You can download the ZIP file and extract the contents to get the project code.
+
+<a href="../../../../assets/attach/connectors/ms-azure-connector.zip">
+    <img src="../../../../assets/img/connectors/download-zip.png" width="200" alt="Download ZIP">
+</a>
+
+## Deployment
+
+Follow these steps to deploy the exported CApp in the Enterprise Integrator Runtime. 
+
+{!references/connectors/deploy-capp.md!}
+
+## Testing
+
+Invoke the API as shown below using the curl command. Curl Application can be downloaded from [here] (https://curl.haxx.se/download.html).
+
+1. Creating a new container in Microsoft Azure Storage for store employee details.
+ 
+   **Sample request**
+
+   `curl -v POST -d {"containerName":"employeedetails"} "http://localhost:8290/resources/createcontainer" -H "Content-Type:application/json"`
+
+   **Expected Response**
+   `{
+       "success": true
+    }`
+    
+2. Retrieve information about the created containers.    
+   
+   **Sample request**
+   
+   `curl -v POST -d {} "http://localhost:8290/resources/listcontainer" -H "Content-Type:application/json"`
+   
+   **Expected Response**
+   
+     It will retrieve all the existing container names.
+     
+     `{
+         "result":{
+            "container":[
+               "employeedetails",
+               "employeefinancedetails"
+            ]
+         }
+      }`
+
+3. Upload text or binary employee data (blob).
+
+   **Sample request**
+   
+     `curl -v POST -d {"containerName": "employeedetails","fileName": "sample.txt","filePath": "/home/kasun/Documents/MSAZURESTORAGE/sample.txt"} "http://localhost:8290/resources/adddetails" -H "Content-Type:application/json"`
+    
+   **Expected Response**
+     
+     `{
+          "success": true
+      }`
+ 
+4. Retrieve information about the uploaded text or binary employee data (blob).
+
+   **Sample request**
+     
+     `curl -v POST -d {"containerName": "employeedetails"} "http://localhost:8290/resources/listdetails" -H "Content-Type:application/json"`
+
+   **Expected Response**
+   
+     It will retrieve the uploaded text or binary name and the file path.
+     
+     `{
+          "result": {
+              "blob": "http://eiconnectortest.blob.core.windows.net/employeedetails/sample.txt"
```

Comment:
You can use this url to get the actual content of the file? If so, please mention it. You can even use WSO2 EI itself to read the file.

Reply:
Shall we implement this as well?

Reply:
updated.